scripts/funcTests/runFuncTests.sh

```diff
@@ -89,6 +86,30 @@ done
 
 # Display .NET CLI info
 $DOTNET --info
+if (( $? )); then
+	echo "DOTNET --info failed!!"
+	exit 1
+fi
+
+# Get recommended version for bootstrapping testing version
```

Comment:
recommended?

Reply:
Changed the comment into `# Install .NET 5 runtimes and .NETCoreapp3.1 runtimes`